chromium_src/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc

```diff
@@ -135,20 +135,32 @@ class PermissionLifetimeCombobox : public views::Combobox,
                << (lifetime ? lifetime->InSeconds() : -1);
     // TODO(https://github.com/brave/brave-browser/issues/14126): Set the
```

Comment:
do you need the TODO ?

Reply:
no, this TODO is implemented in this PR. just forgot to remove the comment.